blog/zh/blog/2022/07/21/weekly-report-0715.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "社区双周报 7.1 - 7.15"
+keywords: 
+- Apache APISIX
+- API 网关
+- 社区周报
+- 贡献者
+description: “独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的每周进展，方便大家参与到 Apache APISIX 社区中来。
+tags: [Events]
+---
+
+> 从 7.1 - 7.15，有 28 位开发者为 Apache APISIX 提交了 99 个 commits。感谢这些小伙伴为 Apache APISIX 添砖加瓦，是你们的无私付出，让 Apache APISIX 项目变得更好！
+
+<!--truncate-->
+
+## 导语
+
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+
+我们还整理了一些适合新来社区的小伙伴们参加的 issue！感兴趣的同学们，走过路过不要错过！
+
+## 贡献者统计
+
+![本周贡献者名单](https://static.apiseven.com/2022/weeklyreport/0721/3.jpg)
+
+![本周新晋贡献者](https://static.apiseven.com/2022/weeklyreport/0721/1.png)
```

Comment:
Borbur already started contributed on or before June 30th, see https://apisix.apache.org/blog/2022/06/21/weekly-report-0621/#contributor-statistics.

Why is he listed as 'New contributor' here?